tests/dictionaries_tests.cpp

```diff
@@ -0,0 +1,190 @@
+#define BOOST_TEST_MODULE RadProtoDictionariesTests
+
+#include "radproto/dictionaries.h"
+#include "radproto/error.h"
+#include <string>
+#include <map>
+#include <cstdint> //uint8_t, uint32_t
+#include <stdexcept>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wold-style-cast"
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#pragma GCC diagnostic ignored "-Wsign-compare"
+#pragma GCC diagnostic ignored "-Wparentheses"
+#include <boost/test/unit_test.hpp>
+#pragma GCC diagnostic pop
+
+BOOST_AUTO_TEST_SUITE(DictionariesTests)
+
+BOOST_AUTO_TEST_SUITE(BasicDictionaryTests)
+
+BOOST_AUTO_TEST_CASE(FunctionAdd)
```

Comment:
I would call this test 'TestAdd'. 'FunctionAdd' in English does not mean "a function called 'add'", it means either addition of functions or something 'functional add'. OTOH, 'TestAdd' clearly says that here we test something called 'add'.